dev/run-dependencies.sh

```diff
@@ -13,7 +13,8 @@ here=$(dirname $0)
 
 $here/migrate-local-test.sh
 
-export MONTAGU_ORDERLY_PATH=$(realpath $here/../src/customConfigTests/git)
+export MONTAGU_ORDERLY_PATH=$(realpath $here/../src/app/git)
```

Comment:
This is the path we use when in 'git mode' and actually we need to use it in 'demo mode' as well now, because of the requirement that the database path was used to start orderly with git integration. This actually works fine for browsing demo reports in orderly web, so have removed this step from the instructions in README for switching mode.